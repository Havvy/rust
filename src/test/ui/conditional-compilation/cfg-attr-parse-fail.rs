// Not having the comma after the predicate in `cfg_attr` is an error.

#[cfg_attr(all())] //~ error: expected `,`, found `)`
struct CfgAttrNoCommaAfterConfigurationPredicate;

#[cfg_attr()] //~ error: expected identifier, found `)`
struct CfgAttrNoConfigurationPredicate;