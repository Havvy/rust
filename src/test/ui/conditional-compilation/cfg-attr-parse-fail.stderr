error: expected `,`, found `)`
  --> $DIR/cfg-attr-parse-fail.rs:3:17
   |
LL | #[cfg_attr(all())] //~ error: expected `,`, found `)`
   |                 ^ expected `,`

error: expected identifier, found `)`
  --> $DIR/cfg-attr-parse-fail.rs:6:12
   |
LL | #[cfg_attr()] //~ error: expected identifier, found `)`
   |            ^ expected identifier

error: aborting due to 2 previous errors

